index.js

```diff
@@ -114,6 +114,16 @@ function SendStream(req, path, options) {
     ? normalizeList(opts.extensions, 'extensions option')
     : []
 
+  if (Array.isArray(opts.precompressed)) {
+    this._precompressionFormats = opts.precompressed
+  } else if (opts.precompressed) {
+    this._precompressionFormats = [{encoding: 'br', extension:'.br'}, {encoding:'gzip', extension:'.gz'}]
```

Comment:
Ah, that's what I was afraid of after reading the specs. It's possible they may remove the framing at some point, but currently it is in there, and it would probably be really confusing to users how to make a `.br` file that this module can use, since eventually tools will exist to make `.br` files and they will contain the file framing.